build/pack.ps1

```diff
@@ -34,11 +34,31 @@ function Pack-One() {
     }
 }
 
+function Pack-One-Dotnet() {
```

Comment:
Minor PowerShell style note, but cmdlet names generally follow the form `Verb-Noun`, where both `Verb` and `Noun` follow PascalCase.
```suggestion
function Pack-OneDotnet() {
```

Reply:
renamed